index.html

```diff
@@ -39,5 +39,189 @@ <h2 class="social order">LET'S CONNECT</h2>
+          <div class="action">
+            <button class="btn">See Project</button>
+          </div>
+        </div>
+      </div>
+    
+      <div class="section3">
+        <div class="right-block">
+          <img class="image-section3" src="./images/Snap3.png" alt="project image">
+        </div>
+        <div class="left-block">
+          <div class="experience">
+            <h1 class="section-title">Tonic</h1>
+            <h1 class="section-titlef">Tonic</h1>
+            <ul class="list">
+              <li class="list1">CANOPY</li>
+              <li class="list2">Back End Dev</li>
+              <li class="list3">2015</li>
+            </ul>
+          </div>
+          <p class="section-description0">A daily selection of privately personalized reads; no
+            accounts or sign-ups
+            required.</p>
+          <ul class="tags">
+            <li class="li11">
+              <article class="text1">html</article>
+            </li>
+            <li class="li12">
+              <article class="text2">css</article>
+            </li>
+            <li class="li13">
+              <article class="text3">javascript
+            </li>
+          </ul>
+          <div class="action">
+            <button class="btn">See Project</button>
+          </div>
+        </div>
+      </div>
+    
+      <div class="section4">
+        <div class="right-block">
+          <img class="image-section4" src="./images/Snap4.png" alt="project image">
+        </div>
+        <div class="left-block">
+          <div class="experience1">
+            <h1 class="section-title1">Multi-Post
+              Stories</h1>
+            <ul class="list">
+              <li class="list1">CANOPY</li>
+              <li class="list2">Back End Dev</li>
+              <li class="list3">2015</li>
+            </ul>
+          </div>
+          <p class="section-description0">A daily selection of privately personalized reads; no
+            accounts or sign-ups
+            required.</p>
+          <ul class="tags">
+            <li class="li11">
+              <article class="text1">html</article>
+            </li>
+            <li class="li12">
+              <article class="text2">css</article>
+            </li>
+            <li class="li13">
+              <article class="text3">javascript
+            </li>
+          </ul>
+          <div class="action">
+            <button class="btn">See Project</button>
+          </div>
+        </div>
+      </div>
+    
+      <div class="section5">
+        <div class="top">
+          <h3 class="about-heading">About Myself</h3>
+          <p class="about-description">I'm a software developer! I can help you build a product, feature or website. Look
+            through some of my work and experience. If you like what you see and have a project you need coded, don't
+            hestiate to contact me.</p>
+          <h3 class="social order2">LET'S CONNECT</h3>
+          <ul class="social-links">
```

Comment:
This is Figma design:
![image](https://user-images.githubusercontent.com/61361037/212447979-e8c301ff-c936-4902-8dff-f3ea7beac5a9.png)

This is your current design:
![image](https://user-images.githubusercontent.com/61361037/212448000-69e152da-052e-45f1-b26c-d9afa6d47139.png)

- In your design, there is a larger gap than the FIGMA design gap.
Try to be as closer to FIGMA design as you can.